pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gnome3
+, gtk-engine-murrine
+, gdk-pixbuf
+, librsvg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "2019-11-13";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "gtk-theme";
+    rev = "8ddf0fcad13c6a3f4fab2772b45561d74a8b4cce";
+    sha256 = "0byzxgd52291vsvwydafnq0xmnnggaryr9c9hfq5hxrd0rr08ibs";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    sassc
+    gtk3
+    inkscape
+    optipng
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+    librsvg
+  ];
+
+  propagatedUserEnvPkgs = [
+    gnome3.gnome-themes-extra
```

Comment:
Why does it need the extra themes?

Reply:
Not needed anymore, when working on 3.1.2 version it was required to have some elements displaying correctly (probably related to the fact that the latest stable version was completly out of date).